lib/cjs/generated/rum.d.ts

```diff
@@ -749,6 +749,34 @@ export declare type RumViewEvent = CommonProperties & {
         };
         [k: string]: unknown;
     };
+    /**
+     * Display properties
+     */
+    display?: {
+        /**
+         * Scroll properties
+         */
+        readonly scroll?: {
+            /**
+             * Distance between the top and the lowest point reached on this view (in pixels)
+             */
+            readonly max_depth?: number;
+            /**
+             * Page scroll height (total height) when the maximum scroll depth was reached for this view (in pixels)
+             */
+            readonly max_depth_scroll_height?: number;
+            /**
+             * Page scroll top (scrolled distance) when the maximum scroll depth was reached for this view (in pixels)
+             */
+            readonly max_depth_scroll_top?: number;
+            /**
+             * Duration between the view start and the scroll event that reached the maximum scroll depth for this view (in nanoseconds)
+             */
+            readonly max_depth_time?: number;
```

Comment:
Shouldn't these attribute be required?

Reply:
oops, I forgot to add them back, thanks for catching